src/gui/editors/AutomationEditor.cpp

```diff
@@ -1278,11 +1121,11 @@ void AutomationEditor::paintEvent(QPaintEvent * pe )
 				y =  yCoordOfLevel( (float)level );
 				if( level % 10 == 0 )
 				{
-					p.setPen( beatLineColor() );
+					p.setPen(m_beatLineColor);
```

Comment:
These are short ifs, combine into single line.

Reply:
That one has an `else` though, I could have the else in the same line as well but I think it will get a bit long. The other alternative is this:

```
if (level % 10 == 0) { p.setPen(m_beatLineColor); }
else { p.setPen(m_lineColor); }
```

Idk, looks a bit odd to me (maybe I'm just a bit used to more visual blocks) but I could have it like that.

Reply:
Nah it's fine, it's normal to see it like that. :)